templates/overrides/search/block--facets-form.html.twig

```diff
@@ -1,12 +1,11 @@
 {#
 /**
  * @file
- * Theme block implementation to display facet form.
+ * Override for blocks with plugin id = 'facets_form'.
  *
- * @see ./core/themes/stable/templates/block/block.html.twig
+ * @see \Drupal\facets_form\Plugin\Block\FacetsFormBlock
```

Comment:
These comments had changed in 1.x since before last rebase.
Please decide if my version or the 1.x version is preferable.